lib/utils/data.js

```diff
@@ -1,4 +1,4 @@
-// Data Resource (files) and Data Package objects (datasets)
+// Data File (files) and Data Dataset objects (datasets)
```

Comment:
This is very minor, but in comments where it was saying Data Package we should replace entire phrase with Dataset. Otherwise as you can see this is wrong.